Modules/process/process_processlist.php

```diff
@@ -155,6 +155,8 @@ public function core_process_list()
         $list[57] = array(_(" * source feed"),ProcessArg::FEEDID,"multiply_by_source_feed",0,DataType::UNDEFINED,"Virtual", 'desc'=>"");
         $list[58] = array(_(" / source feed"),ProcessArg::FEEDID,"divide_by_source_feed",0,DataType::UNDEFINED,"Virtual", 'desc'=>"");
         $list[59] = array(_("1/ source feed"),ProcessArg::FEEDID,"reciprocal_by_source_feed",0,DataType::UNDEFINED,"Virtual", 'desc'=>"");
+        
+        $list[60] = array(_("Reset to ONE"),ProcessArg::NONE,"reset2one",0,DataType::UNDEFINED,"Misc", 'desc'=>"The value '1' is passed back for further processing by the next processor in the processing list.");
```

Comment:
I explain my use case. I have a feed value and I need to check if the feed value is more/less than x value to set other virtual feed to true/false value. The value true/false is for alert activation. Setting to ONE and ZERO the behaviour is as a boolean logic. With boolean virtual feed value, I can show a led alert in a Dashboard.

I attach you the following logic with the process logic:

![reset_to_one](https://cloud.githubusercontent.com/assets/1483414/13346774/00ab6422-dc69-11e5-8fbf-fe3183f7c5e2.PNG)

The logic is possible to use in a input logic and virtual feeds.

![reset_to_one_leds](https://cloud.githubusercontent.com/assets/1483414/13346811/4e6a3dbe-dc69-11e5-8292-b5202b47af16.PNG)

**It is very usefull for binary alerts generation.**